src/main/java/world/reports/NPopulatedCitiesWithinRegion.java

```diff
@@ -8,7 +8,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class NPopulatedCities {
+public class NPopulatedCitiesWithinRegion {
     public static List<City> getCities()
```

Comment:
Whenever you require user input, your function just needs to take a parameter, as in the end, this is going to be called from the UI. 
So in this case, it should be something like
```suggestion
    public static List<City> getNPopulatedCountriesInRegion(int numberOfRows)
```